### ISSUE TITLE
Edit: Create new speech recognition troubleshooting page

### ISSUE THREAD

**Issue by taylojill** [department-of-veterans-affairs/va.gov-team#104072]
## Issue Description
Per Slack DM, Jamie (accessibility SME) has content to be published on the website. See message below:

"I have content that id like to turn into a page on the platform site. it would live in Research and design > Research at VA > Remote mediated research > Assistive technology. It's a guide to speech recognition applications (like dragon and voice control) - how they work and how to troubleshoot issues research participants might have. Here’s the draft (the images have alt text already).

Link to draft Word document

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
